src/Facets/StandardizedCallFacet.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import { LibDiamond } from "../Libraries/LibDiamond.sol";
+
+/// @title StandardizedCallFacet Facet
+/// @author LIFI https://li.finance ed@li.finance
+/// @notice Allows calling different facet methods through a single standardized entrypoint
+/// @custom:version 1.0.0
+contract StandardizedCallFacet {
+    /// External Methods ///
+
+    // @notice Make a standardized call to a facet
+    // @param callData The calldata to forward to the facet
+    function standardizedCall(bytes memory callData) external payable {
+        LibDiamond.DiamondStorage storage ds = LibDiamond.diamondStorage();
```

Comment:
I would propose to call the facetAddress function of the DiamondLoupeFacet here since this is duplicated code from this facet and the code would be much easier to understand if it's just that one function call

Reply:
Calling the DiamondLoupeFacet would incur more gas as it is an external function call. Diamonds should try to use libraries for any internal logic.

Reply:
Good point. 
A comment would help to make this clearer.